src/Kovan-DssSpell.sol

```diff
@@ -16,36 +16,279 @@
-        DssExecLib.authorize(MCD_VAT, MCD_FLASH);
-        DssExecLib.setChangelogAddress("MCD_FLASH", MCD_FLASH);
-
-        // ---------------------------- Update Chainlog version ----------------------------
-        DssExecLib.setChangelogVersion("1.9.1");
+        // https://vote.makerdao.com/polling/QmRCn7Mh#poll-detail
+        CentrifugeCollateralValues memory RWA003 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x4CCc7fED3912A32B6Cf7Db2FdA1554a9FF574099,
+            GEM: 0xDBC559F5058E593981C48f4f09fA34323df42d51,
+            OPERATOR: 0x45e17E350279a2f28243983053B634897BA03b64,
+            INPUT_CONDUIT: 0x45e17E350279a2f28243983053B634897BA03b64,
+            OUTPUT_CONDUIT: 0x45e17E350279a2f28243983053B634897BA03b64,
+            URN:  0x993c239179D6858769996bcAb5989ab2DF75913F,
+            gemID: "RWA003",
+            joinID: "MCD_JOIN_RWA003_A",
+            urnID: "RWA003_A_URN",
+            inputConduitID: "RWA003_A_INPUT_CONDUIT",
+            outputConduitID: "RWA003_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA003",
+            ilk: "RWA003-A",
+            ilk_string: "RWA003-A",
+            ilkRegistryName: "RWA003-A: Centrifuge: ConsolFreight",
+            RATE: SIX_PCT,
+            CEIL: 2 * MILLION,
+            PRICE: 2_359_560 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmQMNfSbGS8qkJbatQgxMUsz27G8YELWgtXeLs8uFCZoY8"
+        });
+
+        // https://vote.makerdao.com/polling/QmRWg6nZ#poll-detail
+        CentrifugeCollateralValues memory RWA004 = CentrifugeCollateralValues({
+            MCD_JOIN: 0xa92D4082BabF785Ba02f9C419509B7d08f2ef271,
+            GEM: 0x146b0abaB80a60Bfa3b4fDDb5056bBcFa4f1fec1,
+            OPERATOR: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            INPUT_CONDUIT: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            OUTPUT_CONDUIT: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            URN:  0xf22C7F5A2AecE1E85263e3cec522BDCD3e392B59,
+            gemID: "RWA004",
+            joinID: "MCD_JOIN_RWA004_A",
+            urnID: "RWA004_A_URN",
+            inputConduitID: "RWA004_A_INPUT_CONDUIT",
+            outputConduitID: "RWA004_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA004",
+            ilk: "RWA004-A",
+            ilk_string: "RWA004-A",
+            ilkRegistryName: "RWA004-A: Centrifuge: Harbor Trade Credit",
+            RATE: SEVEN_PCT,
+            CEIL: 7 * MILLION,
+            PRICE: 8_815_730 * WAD,
+            MAT: 11_000,
+            TAU: 0,
+            DOC: "QmYR2PXwLpdXS8Vp1yS39SPFT1XhmgbsK6XvZ9ApRpNV8M"
+        });
+
+        // https://vote.makerdao.com/polling/QmZCPDVY#poll-detail
+        CentrifugeCollateralValues memory RWA005 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x1233d0DBb55A4Bb41D711d4B584f8DDB15A2Ff88,
+            GEM: 0xcB2A48D26970eE7193d66BAc6F1b3090f2E8f82B,
+            OPERATOR: 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319,
```

Comment:
And, turns out this one is pointing at the `URN` from RWA004 (0xf22c7f5a2aece1e85263e3cec522bdcd3e392b59) instead of 0xdB9f0700EbBac596CCeF5b14D5e23664Db2A184f.

Reply:
Guess I swapped the urns of RWA004 and RWA005 😬 Just fixed this one as well. Thanks for double checking it!